src/views/admin/PersonImportView.vue

```diff
@@ -49,7 +49,7 @@
     object({
       selectedSchule: string().required(t('admin.import.rules.schule.required')),
       selectedRolle: string().required(t('admin.import.rules.rolle.required')),
-      selectedFiles: mixed().required(t('admin.import.rules.files.required')),
+      selectedFiles: array().of(mixed()).length(1, t('admin.import.rules.files.required')),
```

Comment:
Vee validate wird immer noch nicht korrekt getriggert für die Datei, ausser wir laden die Datei hoch und dann löschen die wieder.

 `if (selectedSchule.value === undefined || selectedRolle.value === undefined || !selectedFiles.value?.length) {`  in uploadFile() hilft und lasst uns submitten ohne Datei aber trotzdem wäre es besser, wenn vee validate auch den Fehler  immer zeigt.

Reply:
done